packages/service-library/src/servicelib/docker_constants.py

```diff
@@ -1,3 +1,7 @@
 from typing import Final
 
 PREFIX_DYNAMIC_SIDECAR_VOLUMES: Final[str] = "dyv"
+
+SUFFIX_EGRESS_PROXY_NAME: Final[str] = "egress"
+
+DEFAULT_USER_SERVICES_NETWORK_NAME: Final[str] = "back----end"
```

Comment:
that is an interesting name. any particular reason?

Reply:
add comment why the name in code

Reply:
put a test to not allow to go over certain length, 64 chars